.github/ISSUE_TEMPLATE/01--issues-with-coding-challenges.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Report issues with coding challenges & curriculum
+about: Report issues with a specific challenge, like broken tests, unclear instructions, etc.
+title: ''
+labels: 'scope: curriculum, type: bug, status: waiting triage'
+assignees: ''
+---
+
+<!--
+NOTE: If you're reporting a security issue, don't create a GitHub issue. Instead, email security@freecodecamp.org. We will look into it immediately.
+-->
+
+**Affected page**
+Add a link to the coding challenge with the problem.
+
+
+
+**Your code**
+Copy and paste the code from the editor that you used in between the back-ticks below:
+
+```
+
+
+
+
+```
+
+**Expected behavior**
+Add a clear and concise description of what you expected to happen.
+
+
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. you can drag and drop, png, jpg, gif, etc. in this box.
+
+
+
+**Smartphone (please complete the following information):**
```

Comment:
```suggestion
**System (please complete the following information):**
```

There might be a better word than `System` but I'm not sure that `Smartphone` fits here. 🙂